test/extended/cli/debug.go

```diff
@@ -20,17 +20,29 @@ var _ = g.Describe("[cli] oc debug", func() {
 		err = oc.Run("create").Args("-f", templatePath).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
 
-		exutil.WaitForAnImageStreamTag(oc, oc.Namespace(), "busybox", "latest")
+		exutil.WaitForAnImageStreamTag(oc, oc.Namespace(), "nodejs-ex", "latest")
 		o.Expect(err).NotTo(o.HaveOccurred())
 	})
 
-	g.It("should print the container entrypoint/command", func() {
+	g.It("should print the imagestream-based container entrypoint/command", func() {
+		out, err := oc.Run("debug").Args("dc/nodejs-ex1").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("Debugging with pod/nodejs-ex1-debug, original command: container-entrypoint /usr/libexec/s2i/run\n"))
+	})
+
+	g.It("should print the overridden imagestream-based container entrypoint/command", func() {
+		out, err := oc.Run("debug").Args("dc/nodejs-ex2").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("Debugging with pod/nodejs-ex2-debug, original command: foo bar baz qux\n"))
```

Comment:
these are arguments for the container which are overridden in the deploymentconfig.